File: common-client-api-libs/src/main/java/eu/getsoftware/hotelico/clients/infrastructure/menu/dto/MenuItemDTO.java
package eu.getsoftware.hotelico.clients.infrastructure.menu.dto;

import eu.getsoftware.hotelico.common.dto.BasicDTO;
import lombok.Getter;
import lombok.With;
import lombok.experimental.SuperBuilder;

import java.sql.Timestamp;

@With
@Getter
@SuperBuilder
public class MenuItemDTO extends BasicDTO{

     Long  orderId = 0L;

     int  orderIndex = 0;

     Long  hotelId  = 0L;

     long  cafeId = 0;

     long  senderId  = 0;

     boolean  delimiter = false;

     double  price = 0.0;

     int  amount = 0;

     String  title = "";

     String  shortDescription = "";

     String  description = "";

     String  pictureUrl = null;

     String  previewPictureUrl  = null;

     Timestamp  timestamp = null;
}

File: hotel-app/src/main/kotlin/eu/getsoftware/hotelico/menu/infrastructure/dto/MenuItemDTO.kt
package eu.getsoftware.hotelico.menu.infrastructure.dto;

import eu.getsoftware.hotelico.clients.infrastructure.dto.BasicDTO
import java.sql.Timestamp

class MenuItemDTO: BasicDTO {

     constructor() : super()

     var  orderId = 0L

     var  orderIndex = 0

     var  hotelId: Long? = 0L

     var  cafeId = 0

     var  senderId: Long? = 0

     var  delimiter = false

     var  price = 0.0

     var  amount = 0

     var  title = ""

     var  shortDescription = ""

     var  description = ""

     var  pictureUrl: String? = null

     var  previewPictureUrl: String? = null

     var  timestamp: Timestamp? = null
}
